src/components/PasswordSetterBox.js

```diff
@@ -42,15 +43,26 @@ class PasswordSetterBox extends Nimiq.Observable {
 
     /**
      * @param {?HTMLFormElement} [$el]
-     * @param {object} options
+     * @param {{bgColor: string, buttonI18nTag: string}} options
      * @returns {HTMLFormElement}
      */
     static _createElement($el, options) {
         $el = $el || document.createElement('form');
         $el.classList.add('password-box', 'actionbox', 'setter', `nq-${options.bgColor}-bg`);
 
+        // To enable i18n validation with the dynamic nature of the password box's contents,
+        // all possible i18n tags and texts have to be specified here in the below format to
+        // enable the validator to find them with its regular expression.
         /* eslint-disable max-len */
-        $el.innerHTML = TemplateTags.noVars`
+        /** @type {{[i18nTag: string]: string}} */
+        const buttonVersions = {
+            'passwordbox-confirm': '<button class="submit show-in-repeat" data-i18n="passwordbox-confirm">Confirm</button>',
+            'passwordbox-confirm-create': '<button class="submit show-in-repeat" data-i18n="passwordbox-confirm-create">Create Account</button>',
+            'passwordbox-confirm-log-in': '<button class="submit show-in-repeat" data-i18n="passwordbox-confirm-log-in">Log in Account</button>',
```

Comment:
Is this wording coming from Tammo? I don't feel this is correct english.
```suggestion
            'passwordbox-confirm-log-in': '<button class="submit show-in-repeat" data-i18n="passwordbox-confirm-log-in">Log into Account</button>',
```